replication-api/src/main/java/org/codice/ditto/replication/api/persistence/ReplicatorHistoryManager.java

```diff
@@ -25,7 +25,7 @@
    * @param replicatorId the replication configuration id
    * @return the {@link ReplicationStatus} associated with the given
    * @throws ReplicationPersistenceException if an error occurs while trying to retrieve the object
-   * @throws NotFoundException if a {@link ReplicationStatus} with the given replicator id cannot be
+   * @throws NullPointerException if a {@link ReplicationStatus} with the given replicator id cannot be
```

Comment:
This should be `NotFoundException` right?

Reply:
will fix right now